src/node_options.cc

```diff
@@ -71,6 +80,12 @@ void PerProcessOptions::CheckOptions(std::vector<std::string>* errors) {
       use_largepages != "silent") {
     errors->push_back("invalid value for --use-largepages");
   }
+
+  if (max_worker_thread_count <= 0) {
+    errors->push_back("--max-worker-threads must be a number "
+                      "greater than zero");
```

Comment:
I would allow and document either `0` or `-1` as a value that allows turning this warning off completely, and probably even make that the default behavior.

Reply:
+1 on using `0` as a "turn this off" but not convinced having it off should be the default behavior.